src/csvcubedmodels/rdf/dcat.py

```diff
@@ -119,6 +119,60 @@ def __init__(self, uri: str):
         Resource.__init__(self, uri)
         self.rdf_types.add(DCAT.Dataset)
 
+#The new class would be created here which also inherits the Resource class
+class Distribution(Resource):
+
+    isDistributionOf: Ann[
+        str, Triple(DCAT.isDistributionOf, PropertyStatus.recommended, URIRef)
+    ]
+    #these two will need changing probably
+    access_service: Ann[str, Triple(DCAT.accessService, PropertyStatus.mandatory, URIRef)]
```

Comment:
I don't think any of the `mandatory` triples here should actually be mandatory. I think `title` should be mandatory but that's about it.

Reply:
okay I kept the title mandatory and the rest is recommended